src/test/java/org/mercycorps/translationcards/txcmaker/task/VerifyDeckTaskTest.java

```diff
@@ -161,4 +164,11 @@ public void shouldDownloadAudioFiles() throws Exception {
 
         verify(driveService).downloadAudioFiles(drive, audioFileMetaData, SESSION_ID);
     }
+
+    @Test
+    public void shouldVerifyUsingService() throws Exception {
+        verifyDeckTask.verifyDeck(request);
+
+        verify(verifyDeckService).verify(drive, deck, AUDIO_DIR_ID);
```

Comment:
yo dawg i heard you like verifying. So I verified that you are using the verify service to verify.